core/Opaque.carp

```diff
@@ -0,0 +1,41 @@
+(doc Opaque "The opaque type is an uninhabited type with no constructors.
```

Comment:
Could we load this file in `docs/core/generate_core_docs.carp` and add it to the documentation generation?

Reply:
Done, thanks! And I also re-generated the docs. I always forget to do that -.-